desktop/apps/artwork/components/commercial/templates/inquire.jade

```diff
@@ -52,3 +52,17 @@ if artwork.is_inquireable
       data-context_type='sidebar'
     )
       | Contact #{artwork.partner.type}
+
+else if !artwork.is_for_sale && !artwork.is_acquireable
```

Comment:
I just noticed the screenshot @tayapage6 sent over included the 'Contact' button for this artwork.

I don't think that's right- the partner was just a temporary fair partner and should no longer be having the inquiry functionality. Sorry for confusing you @yuki24 !